websocket/polygon.go

```diff
@@ -45,11 +45,7 @@ type Client struct {
 	log Logger
 }
 
-func New(ctx context.Context, config Config) (*Client, error) {
-	if ctx == nil {
-		ctx = context.Background()
-	}
-
+func New(config Config) (*Client, error) {
```

Comment:
i don't think we actually need this context here. users should call Close instead of cancelling via context.